services/src/main/java/org/apache/unomi/services/impl/definitions/DefinitionsServiceImpl.java

```diff
@@ -59,7 +59,17 @@
 
     private BundleContext bundleContext;
     public DefinitionsServiceImpl() {
-
+        // let's add the built-in value types
```

Comment:
I wonder if we should add all the built-in value types programmatically like this or move them all to JSON files. But having a mixture of this doesn't seem coherent.

Reply:
That came from UNOMI-286 that I based my work on, but I will remove that, as we have set property type defined in json. Don't think we need _unknown_ though.

Reply:
Ok yes unknown is not needed let's get rid of it.